src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -2574,6 +2574,9 @@ def test_vmss_scale_in_policy(self, resource_group):
         self.cmd('vmss update -g {rg} -n {vmss} --scale-in-policy OldestVM', checks=[
             self.check('scaleInPolicy.rules[0]', 'OldestVM')
         ])
+        self.cmd('vmss update -g {rg} -n {vmss} --force-deletion true', checks=[
+            self.check('scaleInPolicy.forceDeletion', True)
+        ])
```

Comment:
Why is the yaml file not re-recorded after this test is modified?

Reply:
There is a problem with the service https://github.com/Azure/azure-cli/issues/20301#issuecomment-986541263